torch/functional.py

```diff
@@ -981,6 +984,114 @@ def cdist(x1, x2, p=2., compute_mode='use_mm_for_euclid_dist_if_necessary'):
     else:
         raise ValueError("{} is not a valid value for compute_mode".format(compute_mode))
 
+def atleast_1d(*tensors):
+    r"""
+atleast_1d(input) -> Tensor
+
+View input tensors to tensor with at least one dimension.
```

Comment:
"Returns a 1-dimensional view of each each input tensor with zero dimensions. Input tensors with one or more dimensions are returned as-is."

The wording here is a little tricky because this only returns a view for the zero dim tensors, right? `contiguous()` has a similar property where it returns `self` if the tensor it's called on is already contiguous.